examples/dapperish.go

```diff
@@ -12,7 +12,9 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/opentracing/api-golang/examples/dapperish"
+	// `go imports` was stumbling over dapperish since it's both a package name
+	// and a source file.
+	dapperish "github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
Hmm. It should only be removing the import if `dapperish` isn't used in the code anywhere.. In general I think we should avoid such hacks / workarounds for bugs in tools; but since this is in the examples / unrelated to the main project I think it's OK to leave it as is